controllers/control/suite_test.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package control
```

Comment:
We don't use ginkgo in k8ssandra-operator. You need to use the testEnv structure of k8ssandra-operator (including multi-cluster stuff), this code can't be used.

Reply:
My bad, I just committed the scaffolded code here without checking. I'll revisit it.